subsys/bluetooth/controller/ll_sw/coex_ticker.c

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+
+#include <zephyr.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+
+#include <logging/log.h>
+#include <drivers/gpio.h>
+#include <sys/__assert.h>
+#include <bluetooth/hci.h>
+#include "controller/hal/ticker.h"
+#include "controller/ticker/ticker.h"
+#include "controller/include/ll.h"
+
+LOG_MODULE_REGISTER(coex_ticker);
+
+#define COEX_RADIO_WORK_DELAY_US  50
+#define COEX_RADIO_WORK_RESCHEDULE_DELAY_US  200
+
+struct coex_ticker_config {
+	struct gpio_dt_spec inhibit_spec;
+	size_t inhibit_delay_us;
+};
+struct coex_ticker_data {
+	const struct device *dev;
+	struct gpio_callback inhibit_irq_cb;
+};
+
+static inline int _ticker_stop(uint8_t inst_idx, uint8_t u_id, uint8_t tic_id)
+{
+	int ret = ticker_stop(inst_idx, u_id, tic_id, NULL, NULL);
+
+	if (ret != TICKER_STATUS_SUCCESS &&
+		ret != TICKER_STATUS_BUSY) {
+		__ASSERT(0, "Failed to stop ticker.\n");
+	}
+
+	return ret;
+}
```

Comment:
Use the contents directed in the caller.

```suggestion
static inline int _ticker_stop(uint8_t inst_idx, uint8_t u_id, uint8_t tic_id)
{
	int ret = ticker_stop(inst_idx, u_id, tic_id, NULL, NULL);

	if (ret != TICKER_STATUS_SUCCESS &&
		ret != TICKER_STATUS_BUSY) {
		__ASSERT(0, "Failed to stop ticker.\n");
	}

	return ret;
}
```